composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.3.7",
         "christian-riesen/base32": "~1.0",
-        "simplesoftwareio/simple-qrcode" : "1.3.*"
+        "bacon/bacon-qr-code": "1.0.*"
```

Comment:
Yes it does. But this could also be added to suggest if that one is merged.
But this package doesn't require ext-gd or illuminate/support (only ext-iconv), so it might be less of an issue.
Still okay with moving this to suggest though.